flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -189,10 +194,6 @@ under the License.
 					<groupId>org.apache.commons</groupId>
 					<artifactId>commons-dbcp2</artifactId>
 				</exclusion>
-				<exclusion>
-					<groupId>com.esri.geometry</groupId>
-					<artifactId>esri-geometry-api</artifactId>
```

Comment:
Why we have to depend on this dependency now?

Reply:
This is a required dependency now, the calcite code can not compile without this jar.